4337/contracts/Safe4337Module.sol

```diff
@@ -84,7 +88,7 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
      * @param data Data payload of the user operation.
      * @param operation Operation type of the user operation.
      */
-    function executeUserOp(address to, uint256 value, bytes memory data, uint8 operation) external {
+    function executeUserOp(address to, uint256 value, bytes calldata data, uint8 operation) external {
```

Comment:
i removed allocating memory for data from the execution functions, saved ~200 gas

Reply:
Good catch, and makes sense - it would have to copy the `memory` bytes when encoding the Safe `execTransactionFromModuleReturnData` (which I _think_ currently involves a loop since there is no `MCOPY` instruction) instead of just doing a `CALLDATACOPY`.